.travis.yml

```diff
@@ -1,17 +1,9 @@
 language: php
 
-# We require precise here, as 5.3 doesn't exist on trusty.
-dist: precise
```

Comment:
This will jump all the way up to xenial, not trust.  Should we be explicit?

The default mysql version will be 5.7.

Reply:
You can add more commits to do that, or open a 2nd PR with your suggestions.
We can merge whatever becomes green first :)

Reply:
I don't see why I would add commits or a PR when I don't know what I'm trying to change.